docs/package/gluon-radvd.rst

```diff
@@ -0,0 +1,36 @@
+gluon-radvd
+===========
+
+This package provides service files and configuration endpoints for the IPv6
+router advertisement daemon.
+
+Arguments
+---------
+
+This package provides skeleton service files for ``uradvd``.
+It requires another package to provide the arguments ``uradvd`` is supposed to
+be launched with.
+
+:doc:`gluon-mesh-batman-adv` and ``gluon-mesh-layer3-common`` are two packages
+providing such arguments.
+
+This allows to have ``uradvd`` announce default routes for layer three meshes,
+while only announcing prefixes for layer two meshes.
+
+site.conf
+---------
+
+radvd.preferred_lifetime : optional
+    - the span of time during which the address can be freely used as a source
+      and destination for traffic. Should be less or equal valid-lifetime.
```

Comment:
```suggestion
      and destination for traffic. Should be less or equal valid_lifetime.
```

Probably makes sense to use the exact spelling of the other option.

Reply:
The texts currently match their description in `uradvd`.
If this is a strong opinion, we can remove the dash in both docs,
but I'd just leave it as it is.

Reply:
(I would argue that gluon-radvd is an abstraction layer and therefore should reference itself where possible. But this really isn't something i want to spend time on arguing. So fine with me. :) )